api/envoy/api/v2/listener/quic_config.proto

```diff
@@ -31,3 +32,20 @@ message QuicProtocolOptions {
   // 20000ms if not specified.
   google.protobuf.Duration crypto_handshake_timeout = 3;
 }
+
+message ActiveQuicListenerConfig {
```

Comment:
I would probably just name this `QuicListenerConfig`. I think we can drop the Active (having it in the other place is a mistake IMO).

In terms of the shutoff, what I'm proposing is just having a `RuntimeFeatureFlag` which says whether to process packets or not. I think this is ultimately the kill switch we are looking for? @alyssawilk @danzh2010  WDYT?

Reply:
Again I think it's not the switch I'm looking for in the long term (because not granular enough) but I'm fine landing whatever controls we think are useful today and we can add the finicky ones once we're running in prod and care more :-)